src/main/java/graphql/schema/GraphQLCodeRegistry.java

```diff
@@ -202,6 +203,38 @@ private Builder(GraphQLCodeRegistry codeRegistry) {
             this.defaultDataFetcherFactory = codeRegistry.defaultDataFetcherFactory;
         }
 
+        /**
```

Comment:
This allows us to mark a code registry and know if it changes since we marked it.